### DISCUSSION TITLE
美國大學生開發「GPTZero」應用程式，有效檢查文章是否由「ChatGPT」生成

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#676] [General]
美國普林斯頓大學（Princeton University）大四學生愛德華．田（Edward Tian） 開發一款應用程式「[GPTZero](https://gptzero.me/)」，能夠有效地檢查文章是否為AI程式ChatGPT撰寫，「讓AI現形。」

GPTZero的運作：GPTZero已經先受過資料庫訓練，並採取「困惑度」和「突發性（burstiness）」兩項檢測指標。如果 GPTZero 對文本感到困惑，那麼它具有很高的複雜性並且更有可能是人為編寫的。然而，如果文本對機器人來說更熟悉（因為它已經接受過此類數據的訓練）那麼它的複雜性就會很低，因此更有可能是由 AI 生成的；另外，突發性比較句子的變化，人類傾向於以更大的突發性寫作，例如一些較長或複雜的句子與較短的句子一起使用，但AI 的句子往往更統一。

ChatGPT之餘GPTZero就猶如駭客與資安長的存在，大家可以試試看這幾位大學生的GPTZero！

文章連結：https://www.thenewslens.com/article/179466

**Comment by 0212Jerry** [comment]
這是一個很好的發明，讓Ai 辨識Ai ，這對於審查論文、資料以及比賽的判別非常有用。但這還是隱藏風險，因為我們知道Ai 不斷的在進步，總有一天可以寫出更像人類的文章，我們必須確保這套辨別系統可以凌駕於待審查的Ai 之上，否則使用者反而更容易被混淆。
